odoo/addons/base/models/ir_translation.py

```diff
@@ -171,7 +186,9 @@ class IrTranslation(models.Model):
                               ('translated', 'Translated')],
                              string="Status", default='to_translate',
                              help="Automatically set to let administators find new terms that might need to be translated")
-
+    # FIXME: index these? Maybe index (web, frontend) or something?
+    web = fields.Boolean()
+    frontend = fields.Boolean()
```

Comment:
`web` implies `code` and `frontend` implies `web`. At least I think it is but worth fixing as could simplify the code.

Reply:
FIxing what?

Reply:
Frenglish, I mean document it, add constraint/onchange/inverse method/add magic to ensure the consistency.